.github/PULL_REQUEST_TEMPLATE/manual_maintenance.md

```diff
@@ -11,7 +11,7 @@ creation URL (or simply copy this md file into the description)
 
 The following known issues need to be reviewed in case a compatible version is available. Add new known issues as they
 appear.
-
+- [ ] Update node.js to version 20.6.1 breaks the build
```

Comment:
This got updated to 20.6.2

Reply:
In keycloak, my bad

Reply:
@fsoffel only @types/node was updated not node itself, I updated because updatepackages suggested it and nothing broke, I can downgrade to previous if needed.

Reply:
Where would we update node?

Reply:
dockerFile

Reply:
@fsoffel see docker-compose.yml (line 203)

Reply:
Ah, yeah. Thanks. The question stays, should we update @types/node that. I don't really know what it is responsible for. But as long as nothing seems to break, this is fine.